### PR TITLE
feat: Badge/button components

### PR DESCRIPTION
**Pull request by AtulHande03a**
Created badge and button component. 
Screenshot (40)
Screenshot (41)
Screenshot (42)
Screenshot (43)
Screenshot (44)
Screenshot (45)
s

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  background-color: var(--red);
+}
+
+.green-badge {
+  background-color: var(--green);
+}
+
+.border-white {
+  border: 1px solid var(--white);
+}
```
Comment (horsemaker) [inline components/badge/badge.css:92]: Seems like a generic class.

**Review by horsemaker — APPROVED** [review]
LGTM! 👍

File: components/badge/badge.css (hunk 1 above)
Comment (AtulHande03a) [reply components/badge/badge.css:92]: Yes, nice find.. will add this to utility

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
+.btn {
+  width: max-content;
+  display: inline-block;
+  padding: 0.75rem 1rem;
+  min-width: 80px;
```
Comment (poojavishnoi) [inline components/button/button.css:5]: Try to use consistent unit:)

**Review by poojavishnoi — APPROVED** [review]
Looks good..

File: components/button/button.css (hunk 2 above)
Comment (AtulHande03a) [reply components/button/button.css:5]: yes, will fix this.

**Pull request merged**